packages/fluentui/react-icons-northstar/src/components/FilterIcon.tsx

```diff
@@ -5,14 +5,14 @@ import { iconClassNames } from '../utils/iconClassNames';
 
 export const FilterIcon = createSvgIcon({
   svg: ({ classes }) => (
-    <svg role="presentation" focusable="false" viewBox="8 8 18 16" className={classes.svg}>
+    <svg role="presentation" className={classes.svg} viewBox="0 0 20 20">
```

Comment:
Why was the `focusable="false"` removed here?

Reply:
I started removing this attribute because it was only used by IE11, but then realized I should leave it.  I'm reverting.